builders/pallets-precompiles/precompiles/identity.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Identity Precompile
+description: Learn all you need to know about the Identity Precompile, such as its address, Solidity interface, and how to interact with it using popular Ethereum libraries.
+---
+
+# Identity Precompile on Moonbeam
+
+## Introduction {: #introduction }
+
+The Identity Precompile is a Solidity interface that allows you to create, manage, and retrieve information on on-chain identities. Identities are attached to accounts and include personal information, such as your legal name, display name, website, Twitter handle, Riot (now known as Element) name, and more. You can also take advantage of custom fields to include any other relevant information.
+
+The Identity Precompile interacts directly with Substrate's [Identity Pallet](/builders/pallets-precompiles/pallets/identity){target=_blank} to provide the functionality needed to create and manage identities. This pallet is coded in Rust and is normally not accessible from the Ethereum side of Moonbeam. However, the Identity Precompile allows you to access this functionality directly from the Solidity interface.
+
+The Identity Precompile is located at the following address:
+
+=== "Moonbeam"
+
+     ```text
+     {{networks.moonbeam.precompiles.identity }}
+     ```
+
+=== "Moonriver"
+
+     ```text
+     {{networks.moonriver.precompiles.identity }}
+     ```
+
+=== "Moonbase Alpha"
+
+     ```text
+     {{networks.moonriver.precompiles.identity }}
+     ```
+
+--8<-- 'text/precompiles/security.md'
+
+## The Identity Precompile Solidity Interface {: #the-solidity-interface }
+
+[`Identity.sol`](https://github.com/moonbeam-foundation/moonbeam/blob/master/precompiles/identity/Identity.sol){target=_blank} is a Solidity interface that allows developers to interact with the precompile's methods.
+
+??? code "Identity.sol"
+
+    ```solidity
+    --8<-- 'code/precompiles/identity/Identity.sol'
+    ```
+
+The Identity Precompile contains some functions that can be called by anyone and some judgment-related functions that must be called by a registrar. The functions that can be called by anyone are as follows:
```

Comment:
This is a nitpick but would "can only be called by a registrar" be more specific wording than "must be called by a registrar"?

Reply:
Yeahh!! I'm here for it 🙂